app/middleware/parameter-validation.ts

```diff
@@ -22,6 +22,22 @@ const RANGESET_ROUTE_REGEX = /^\/.*\/ogc-api-coverages\/.*\/collections\/.*\/cov
  */
 const validLinkTypeValues = ['http', 'https', 's3'];
 
+/**
+ * Returns the bucket setup instruction 
+ *
+ * @param req - The client request
+ * @param destinationUrl - The destinationUrl
+ * @returns the bucket setup instruction
+ */
+function bucketInstruction(req: HarmonyRequest, destinationUrl: string): string {
+  const bucketPolicyUrl = `${getRequestRoot(req)}/staging-bucket-policy?bucketPath=${destinationUrl}`;
+  const instructions = [`The s3 bucket must be created in the ${awsDefaultRegion} region with 'ACL disabled' which is the default Object Ownership setting in AWS S3.`,
```

Comment:
This is a good place to just split the lines and concatenate using '+' to prevent the long lines.